test/functions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once ('../functions.php');
+
+class FunctionsTest extends TestCase {
+    public function testDisplayFilms() {
+        $arrayData = ['film1' => ['title' => 'A Few Good Men', 'release_year' => 1992, 'my_review' => 4, 'bechdel_status' => 'FAIL'], 'film2' => ['title' => 'Stand By Me', 'release_year' => 1986, 'my_review' => 5, 'bechdel_status' => 'FAIL']];
```

Comment:
Is $arrayData representative of the data the you receive from the database? Isn't it more likely to be:

`$arrayData = [['title' => 'A Few Good Men', 'release_year' => 1992, 'my_review' => 4, 'bechdel_status' => 'FAIL'], ['title' => 'Stand By Me', 'release_year' => 1986, 'my_review' => 5, 'bechdel_status' => 'FAIL']];`

Reply:
Yes, thanks. I got confused because I know the fetchAll() is set to bring back an associative array so added in the the keys. Have taken them out now.